samples/drivers/led/pwm/sample.yaml

```
@@ -17,6 +17,6 @@ tests:
         - "Turned on"
         - "Turned off"
         - "Increasing brightness gradually"
-        - "Blinking on: 0.1 sec, off: 0.1 sec"
-        - "(Blinking on: 1 sec, off: 1 sec|Cycle period not supported)"
+        - "Blinking on: 100 msec, off: 100 msec"
```

Since 6a84631eec9619c333d85cb784750ca74c531b23 made these values configurable, it would be better to use:
```suggestion
        - "Blinking on: \\d+ msec, off: \\d+ msec"
```